src/libsystemd/sd-device/device-private.c

```diff
@@ -1001,6 +1001,7 @@ int device_update_db(sd_device *device) {
 
         path = strjoina("/run/udev/data/", id);
 
+#if !UDEV_DB_STORE_ALL
```

Comment:
uh, i am not sure i like this. this changes behaviour of our api calls quite a bit because you never know what sd_device_is_initialized() actually means then.

if you really want this, why not just define a udev rule that matches all devices and sets some dummy property?

Reply:
Let me see how much overhead this has compared to this solution. For now I remove this patch.